### ISSUE TITLE
Render videos with more options for edges

### ISSUE THREAD

**Issue by isabelperezf** [talmolab/sleap#997]
Dear sleap people,

I (with @auesro) am trying to export a full video with customized marker sizes and colors (not doable in the GUI). So far I am using a single video in which I have predicted all frames (Run inference on all video frames):

````
PREDICTIONS_FILE = "labels.vmodel27_entirevideo.slp"
video_file = "8-2-2021_Chloroquine_1.mp4"
import sleap
from sleap.io.video import Video
from sleap.io.visuals import save_labeled_video 
save_labeled_video('model27.mp4',labels=sleap.load_file(PREDICTIONS_FILE),video = Video.from_media(filename=video_file),frames=[100],fps=30,color_manager='five+')
````
However, I am getting the following error:

````
ValueError                                Traceback (most recent call last)
[<ipython-input-12-e70292c9851a>](https://localhost:8080/#) in <module>
      2 from sleap.io.video import Video
      3 from sleap.io.visuals import save_labeled_video
----> 4 save_labeled_video('model27.mp4',labels=sleap.load_file(PREDICTIONS_FILE),video = Video.from_media(filename=video_file),frames=[100],fps=30,color_manager='five+')

[/usr/local/lib/python3.7/dist-packages/sleap/io/visuals.py](https://localhost:8080/#) in save_labeled_video(filename, labels, video, frames, fps, scale, crop_size_xy, show_edges, color_manager, gui_progress)
    503         out_q=q2,
    504         labels=labels,
--> 505         video_idx=labels.videos.index(video),
    506         scale=scale,
    507         show_edges=show_edges,

ValueError: Video(backend=MediaVideo(filename='8-2-2021_Chloroquine_1.mp4', grayscale=False, bgr=True, dataset='', input_format='')) is not in list
````

Any ideas what might be wrong? Is there some missing code to generate the videos?

Thanks

**Comment by roomrys** [comment]
Hi @isabelperezf,

Instead of creating a new video object, you need to reference the video object that is already in the `.slp` project.

```py
PREDICTIONS_FILE = "labels.vmodel27_entirevideo.slp"

import sleap
labels = sleap.Labels.load_file(PREDICTIONS_FILE )
video = labels.videos[video_idx]
...
```

where `video_idx` would be the index of the video object in the `labels.videos` list. If you have just a single video in the project, then `video_idx = 0`. You can list all the videos in the project (and manually find the `video_idx`) with by running `print(labels.videos)`.

Let us know if this helps!

Thanks,
Liezl

**Comment by auesro** [comment]
Hi @roomrys

You are right! Now it works!
We have 2 more questions:
-Is it possible also to change the edges from lines to wedges?
-Is it possible to apply different colors to nodes and edges?

Thanks a lot!

**Comment by roomrys** [comment]
Hi @isabelperezf,

I took a look and it seems we were not passing in any parameters to draw the edges as wedges nor use different colors for the edges. I opened PR #998 which will allow the user to draw wedges and will open another PR to apply different colors to nodes and edges. I will give an update when those are merged so you can get the feature ASAP.

Thanks,
Liezl

**Comment by auesro** [comment]
Thanks a lot @roomrys !
Looking forward to the new option!

**Comment by roomrys** [comment]
Hi @auesro,

You bring up a good point. [SLEAP sets the color manager using the app's settings if using the gui visuals. Otherwise, the color manager is set to `None`](https://github.com/talmolab/sleap/blob/develop/sleap/gui/commands.py#L1200-L1203) and is [subsequently initialized](https://github.com/talmolab/sleap/blob/develop/sleap/io/visuals.py#L190-L192) using the provided `.slp` path ([and its associated config file](https://github.com/talmolab/sleap/blob/develop/sleap/gui/color.py#L40-L44)). 

I can pass in the palette argument to set the color through the CLI and also set which instances to color if I add another argument. `sleap-render` is definitely getting a makeover!

**Comment by roomrys** [comment]
Hi @auesro, @isabelperezf,

I merged #998 into the develop branch. If you would like to get this feature as soon as possible, then you can [install SLEAP from source](https://sleap.ai/installation.html#conda-from-source); otherwise, it will be available in the next release.

Let me know if you have any feedback on this feature and/or other ideas for improvement.

Thanks,
Liezl

**Comment by auesro** [comment]
Thanks @roomrys! 
We will test it as soon as possible!
